reana_client/cli/workflow.py

```diff
@@ -712,11 +712,14 @@ def add_verbose_data_from_response(response, verbose_headers, headers, data):
 @click.option("--json", "json_format", count=True, help="Get output in JSON format.")
 @add_access_token_options
 @click.option(
-    "-s", "--step", "steps", multiple=True, help="Get logs of a specific step."
+    "--filter",
+    "filters",
+    multiple=True,
+    help="Filter the job logs with name=value pairs. Available filters are step, compute_backend, docker_img and status.",
```

Comment:
Please sort alphabetically: c_d, d_i, ste, sta.

Reply:
Hmm, the message "Filter the job logs with name=value pairs" may be a bit cryptic to the end users.  What about making it even more explicit, such as:

```
Filter job logs to include only those steps that match certain filtering criteria. Use --filter name=value pairs. Available...
```

Reply:
Fixed!